tools/workspace-plugin/src/executors/build/lib/swc.ts

```diff
@@ -1,14 +1,20 @@
-import { readFileSync } from 'node:fs';
 import { mkdir, writeFile } from 'node:fs/promises';
-import { basename, dirname, join } from 'node:path';
+import { dirname, join } from 'node:path';
 
 import { globSync } from 'fast-glob';
 import { isMatch } from 'micromatch';
-import { transform, type Config } from '@swc/core';
+import { transformFile, type Config } from '@swc/core';
 import { logger, readJsonFile } from '@nx/devkit';
 
 import { type NormalizedOptions } from './shared';
 
+// extend @swc/core types by missing apis
+declare module '@swc/core' {
+  interface BaseModuleConfig {
+    resolveFully?: boolean;
```

Comment:
Should we create a new issue/PR for @swc/core or @swc/types to fix this?

Reply:
https://github.com/swc-project/swc/issues/9672